### PR TITLE
Turn on Bugzilla verification

### PR DESCRIPTION
**Pull request by stevekuznetsov**
/assign @smarterclayton @eparis @derekwaynecarr

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -1094,3 +1097,25 @@ external_plugins:
...
+      target_release: 4.0.0
+    release-4.0:
+      is_open: true
+      target_release: 4.0.0
+    openshift-4.1:
+      is_open: true
+      target_release: 4.1.0
+    release-4.1:
+      is_open: true
+      target_release: 4.1.0
```
Comment (smarterclayton) [inline cluster/ci/config/prow/plugins.yaml]: Target release is what the bug has to be opened against as a target?  If so, this should be `4.1.z` I think.

File: cluster/ci/config/prow/plugins.yaml [hunk 2]
```diff
@@ -1094,3 +1097,25 @@ external_plugins:
...
+  default:
+    openshift-4.0:
+      is_open: true
+      target_release: 4.0.0
+    release-4.0:
+      is_open: true
+      target_release: 4.0.0
+    openshift-4.1:
+      is_open: true
+      target_release: 4.1.0
```
Comment (smarterclayton) [inline cluster/ci/config/prow/plugins.yaml]: Same as below

File: cluster/ci/config/prow/plugins.yaml [hunk 3]
```diff
@@ -1094,3 +1097,25 @@ external_plugins:
     - name: needs-rebase
       events:
       - pull_request
+
+bugzilla:
+  default:
+    openshift-4.0:
```
Comment (smarterclayton) [inline cluster/ci/config/prow/plugins.yaml]: These are dead, let's just remove these.  There will never be 4.0 code, I deleted most of the CI jobs, it's gone, etc.

**Comment by derekwaynecarr** [comment]
agree with change.

**Comment by stevekuznetsov** [comment]
Need https://github.com/openshift/ci-operator-prowgen/pull/171 to get tests to run

File: cluster/ci/config/prow/plugins.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply cluster/ci/config/prow/plugins.yaml]: done

File: cluster/ci/config/prow/plugins.yaml (hunk 2 above)
Comment (stevekuznetsov) [reply cluster/ci/config/prow/plugins.yaml]: done

File: cluster/ci/config/prow/plugins.yaml (hunk 3 above)
Comment (stevekuznetsov) [reply cluster/ci/config/prow/plugins.yaml]: done

File: cluster/ci/config/prow/plugins.yaml [hunk 4]
```diff
@@ -1094,3 +1097,19 @@ external_plugins:
...
+bugzilla:
+  default:
+    openshift-4.1:
+      is_open: true
+      target_release: 4.1.z
+    release-4.1:
+      is_open: true
+      target_release: 4.1.z
+    # TODO: turn this on when we branch
+    # openshift-4.2:
```
Comment (wking) [inline cluster/ci/config/prow/plugins.yaml]: Let's turn this in now.  Pre-branch PRs should target master and end up in the 4.2 branches via magical ART fast-forwards, so we van block direct-to-4.2 PRs now.

Comment (smarterclayton) [reply cluster/ci/config/prow/plugins.yaml]: There are engineering teams delivering to "master" as openshift-4.2.

Comment (stevekuznetsov) [reply cluster/ci/config/prow/plugins.yaml]: nope, so these are repos like etcd where they do their dev in this branch. not master

**Comment by smarterclayton** [comment]
lgtm, feel free to self tag once you fix the config job

**Comment by stevekuznetsov** [comment]
Needs https://github.com/kubernetes/test-infra/pull/12908 to pass `checkconfig`

**Pull request merged**